mmv1/products/networkmanagement/VpcFlowLogsConfig.yaml

```diff
@@ -153,6 +180,16 @@ properties:
     type: String
     description: |
       Traffic will be logged from the VPN Tunnel. Format: projects/{project_id}/regions/{region}/vpnTunnels/{name}
+  - name: 'subnet'
```

Comment:
Question: Is it possible to specify both `network` and `subnet` together? If so, could you please add a test to include this case, otherwise, we may need to mark them as conflict_with.

Reply:
I believe [`Conflicts`](https://github.com/GoogleCloudPlatform/magic-modules/blob/2a22759ec0725321ddcb6ae69a41d133ef0db933/mmv1/api/type.go#L126C2-L126C11) is what you're looking for. `exactly_one_of` would require one of the fields to be set.

However, since `vpn_tunnel` and `interconnect_attachment` are already implemented and lack of this validation on the Terraform side, now adding it would be a breaking change. I'd suggest we state that at most one of the fields can be specified in the field descriptions and rely on the API-level validation

https://googlecloudplatform.github.io/magic-modules/breaking-changes/breaking-changes/#field-level-breaking-changes

Reply:
I believe [`Conflicts`](https://github.com/GoogleCloudPlatform/magic-modules/blob/2a22759ec0725321ddcb6ae69a41d133ef0db933/mmv1/api/type.go#L126C2-L126C11) is what you're looking for. `exactly_one_of` would require one of the fields to be set.

However, since `vpn_tunnel` and `interconnect_attachment` are already implemented and lack of this validation on the Terraform side, now adding it would be a breaking change. I'd suggest we state that at most one of the fields can be specified in the field descriptions and rely on the API-level validation

https://googlecloudplatform.github.io/magic-modules/breaking-changes/breaking-changes/#field-level-breaking-changes